deployment/aks-periscope.yaml

```diff
@@ -83,7 +85,7 @@ spec:
         - secretRef:
             name: azureblob-secret
         volumeMounts:
-        - mountPath: /aks-periscope
+        - mountPath: /log/aks-periscope
```

Comment:
💡 minor: do we need to document this anywhere?

Reply:
@arnaud-tincelin what is this change for? I recommend not changing this file at all due to production downstream dependencies

Reply:
What dependencies are your referring to?

Another PR about dockerfile (which must be done first) puts the binary at /aks-periscope so we can't mount a volume there anymore.

Anyway, the best option might be first to remove all dependencies to files in the code so that we don't need this volume at all.

Reply:
The az cli and vs code extensions both currently pull and deploy the live raw version of this file, and might make other assumptions about its structure:

https://github.com/Azure/vscode-aks-tools/blob/430e8931a06ca6fe59bc8bff9bb98954106237f8/src/commands/periscope/helpers/periscopehelper.ts#L118

https://github.com/Azure/azure-cli-extensions/blob/b613a52b445ee49509803c29622b4eb664d84501/src/aks-preview/azext_aks_preview/custom.py#L1991

I assume this design is to allow the versions of periscope which are deployed by the CLI / vs code to be updated via changes to the periscope repo only - however I don't particularly agree with it and consider it a break waiting to happen, as well as unnecessary friction and complexity  when trying to change periscope itself - would be happier to have dedicated files like /azcli/aks-periscope.yaml and /vscode/aks-periscope.yaml if they do reside in this repo.

Not sure if any of your planned changes would actually break the integrations, but I suggest treating this as a "golden file" and not touching it for the time being - perhaps the kustomize changes could reside in a deployment/kustomize or deployment/dev subdir for the time being?

Reply:
az cli & vscode are independent from aks-periscope so we are not going to add anything specific for any tool.
What we could do is to git tag the version of the aks-periscope.yaml file before this PR and the PR on dockerfile so that it matches with a docker image tag.
The only change for azcli & vscode would be in the URL where you would no longer target /latest/ (which is a good thing).
This way we can make any changes without fear to break what is currently in use. What do you think?